### PR TITLE
Improve message for target folder

### PR DESCRIPTION
**Pull request by fwilhe**

### CODE REVIEW

File: src/utils/scaffold.ts [hunk 1]
```diff
@@ -19,9 +19,9 @@ export async function shouldBuildScaffold(projectDir: string, buildScaffold: boo
     return false;
   }
 
-  cli.log('This folder does not contain a `package.json`.');
+  cli.log('The target folder (' + projectDir + ') does not contain a `package.json`.');
```
Comment (marikaner) [inline src/utils/scaffold.ts:22]:
```suggestion
  cli.log(`The target directory (${projectDir}) does not contain a \`package.json.\``);
```

File: src/utils/scaffold.ts [hunk 2]
```diff
@@ -19,9 +19,9 @@ export async function shouldBuildScaffold(projectDir: string, buildScaffold: boo
...
 
-  if (await cli.confirm('Should a new `nest.js` project be initialized in this folder? (y|n)')) {
+  if (await cli.confirm('Should a new `nest.js` project be initialized in the target folder? (y|n)')) {
```
Comment (marikaner) [inline src/utils/scaffold.ts:24]:
```suggestion
  if (await cli.confirm('Should a new `nest.js` project be initialized in the target directory? (y|n)')) {
```

File: src/utils/scaffold.ts (hunk 1 above)
Comment (marikaner) [reply src/utils/scaffold.ts:22]: I assume escaping should work like this.

**Comment by fwilhe** [comment]
Thanks for the suggestion Marika, seems to work, tried it out in a node REPL

**Comment by florian-richter** [comment]
@fwilhe Could you please pull the newest changes of the master into your branch so the PR checks will pass?

**Comment by fwilhe** [comment]
> @fwilhe Could you please pull the newest changes of the master into your branch so the PR checks will pass?

Updated.

**Pull request merged**